pallets/permissions/src/weights.rs

```diff
@@ -41,32 +41,32 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	fn add_as_admin() -> Weight {
-		(85_000_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(1 as Weight))
-			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+		(14_000_000 as Weight)
```

Comment:
Did we run this on the bench instance on gcp?

Reply:
is it to be run only on gcp?

Reply:
yes we cannot use out own machines as benchmark env, instead lets use the cloud one that is following the recommended specs.

Reply:
We are running all the benchmarks in a follow up PR when all the pallets are taken into account.